.travis.yml

```diff
@@ -28,43 +28,44 @@ env:
     - VOLATILE_DIR=/tmp
     - DD_CASHER_DIR=/tmp/casher
   matrix:
-    - TRAVIS_FLAVOR=default
-    - TRAVIS_FLAVOR=apache
-    - TRAVIS_FLAVOR=cassandra
-    - TRAVIS_FLAVOR=couchdb
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=0.90.13
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=1.0.3
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=1.1.2
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=1.2.4
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=1.3.7
-    - TRAVIS_FLAVOR=elasticsearch FLAVOR_VERSION=1.4.2
-    - TRAVIS_FLAVOR=etcd
-    - TRAVIS_FLAVOR=fluentd
-    # FIXME: cannot enable gearman on Travis right now
-    # because it needs boost
-    # - TRAVIS_FLAVOR=gearman
-    - TRAVIS_FLAVOR=haproxy
-    - TRAVIS_FLAVOR=lighttpd
-    - TRAVIS_FLAVOR=memcache
-    - TRAVIS_FLAVOR=mongo
-    - TRAVIS_FLAVOR=mysql
-    - TRAVIS_FLAVOR=nginx
-    - TRAVIS_FLAVOR=pgbouncer
-    - TRAVIS_FLAVOR=phpfpm
-    - TRAVIS_FLAVOR=postgres FLAVOR_VERSION="9.0.18"
-    - TRAVIS_FLAVOR=postgres FLAVOR_VERSION="9.1.14"
-    - TRAVIS_FLAVOR=postgres FLAVOR_VERSION="9.2.9"
-    - TRAVIS_FLAVOR=postgres FLAVOR_VERSION="9.3.5"
-    - TRAVIS_FLAVOR=postgres FLAVOR_VERSION="9.4.0"
-    - TRAVIS_FLAVOR=rabbitmq
-    - TRAVIS_FLAVOR=redis FLAVOR_VERSION="2.4"
-    - TRAVIS_FLAVOR=redis FLAVOR_VERSION="2.6"
-    - TRAVIS_FLAVOR=redis FLAVOR_VERSION="2.8"
-    - TRAVIS_FLAVOR=snmpd
-    - TRAVIS_FLAVOR=ssh
-    - TRAVIS_FLAVOR=supervisord
-    - TRAVIS_FLAVOR=sysstat
-    - TRAVIS_FLAVOR=tomcat # JMX testing machine / need the other ones before
+    # - TRAVIS_FLAVOR=default
```

Comment:
Just added a note on PR description. I am still working on this check to add tests, and bypassing other tests